Formula/musikcube.rb

```diff
@@ -0,0 +1,40 @@
+class Musikcube < Formula
+  desc "Terminal-based audio engine, library, player and server"
+  homepage "https://musikcube.com"
+  url "https://github.com/clangen/musikcube/archive/0.99.3.tar.gz"
+  sha256 "584c8225caad46fde83cdd520ef7d4a6633471de9693651f46d520040e9fb51d"
+  license "BSD-3-Clause"
+  head "https://github.com/clangen/musikcube.git", branch: "master"
+
+  depends_on "ccache" => :build
+  depends_on "cmake" => :build
+  depends_on "asio"
+  depends_on "curl"
+  depends_on "ffmpeg"
+  depends_on "game-music-emu"
+  depends_on "gnutls"
+  depends_on "lame"
+  depends_on "libev"
+  depends_on "libmicrohttpd"
+  depends_on "libogg"
+  depends_on "libopenmpt"
+  depends_on "libvorbis"
+  depends_on "ncurses"
+  depends_on "openssl@1.1"
+  depends_on "taglib"
+
+  def install
+    args = std_cmake_args + %w[
+      -DCMAKE_BUILD_TYPE=Release
+    ]
+    system "cmake", ".", *args
+    system "make"
+    system "cmake", "."
+    system "make", "install"
+  end
+
+  test do
+    system "musikcubed", "--start"
+    system "musikcubed", "--stop"
```

Comment:
Can we check any output here? Is there some status command we can run?

Reply:
Updated to ensure the daemon actually spawned correctly by checking for the existence of a lock file. There's not a lot else we can currently do, as musikcube is by and large an interactive frontend app. https://github.com/Homebrew/homebrew-core/pull/118938/commits/e17c6a56dba3558134c0c25863d3c34afbc46406

Reply:
Fixed in subsequent commits.